public/js/src/module/gui.js

```diff
@@ -486,6 +488,9 @@ function printOcForm() {
                         }
                         if ( format.queries === 'yes' ) {
                             printified = $dn.trigger( 'printify.enketo' );
+                            textPrints.forEach( ( textPrint ) => {
+                                textPrint.dispatchEvent( events.PrintifyText() );
```

Comment:
the event has now been renamed to Printify()

Reply:
These events on non-DN questions should always be dispatched (so not only when queries==='yes')

Reply:
code updated